tests/test_client.py

```diff
@@ -86,6 +88,59 @@ def test_get_latest_version(index_client):
     assert latest.hashes == doc.hashes
 
 
+def create_test_index(index_client, version=None, current_did=None):
+    """
+        Shorthand for creating new index entries for test purposes
+        Args:
+            index_client (indexclient.client.IndexClient):
+            version (str): version number of the index should be added
+            current_did (str): if specifed add a version of the index with the did = current_did
+        Returns:
+            indexclient.client.Document: versioned document
+        """
+
+    did = str(uuid.uuid4())
+
+    # add a new node index
+    hashes = {'md5': u'ab167e49d25b488939b1ede42752458c'}
+
+    if current_did:
+        data = {
+            "size": 10,
+            "hashes": hashes,
+            "urls": [],
+            "form": "object",
+            "file_name": did + "_super_indexed_razzmatazz.xtx"
+        }
+        if version:
+            data["version"] = version
+        doc = index_client.add_version(current_did, Document(None, None, data))
```

Comment:
This appears to be two functions smushed into one. One is creating, one is update.